sphinx/ext/intersphinx/_load.py

```diff
@@ -107,15 +107,21 @@ def fetch_inventory_group(
     app: Sphinx,
     now: int,
 ) -> bool:
-    cache_time = now - app.config.intersphinx_cache_limit * 86400
+    if app.config.intersphinx_cache_limit < 0:
+        cache_time = now - app.config.intersphinx_cache_limit * 86400
+    else:
+        cache_time = None
```

Comment:
This would mean that we don't have to change the test below?
```suggestion
        cache_time = 0
```

Reply:
Yes, but the semantics don't seem very clear?

Reply:
fixed.